frontend/src/pages/Customer/CouponList/components/CafeInfo/index.tsx

```diff
@@ -11,24 +11,17 @@ import {
   ProgressBarContainer,
   StampCount,
 } from './style';
+import { Coupon, CouponInfo } from '../../../../../types/domain/coupon';
 
 interface CafeInfoProps {
-  cafeName: string;
-  stampCount: number;
-  maxStampCount: number;
-  isFavorites: boolean;
-  frontImageUrl: string;
+  cafeInfo: Coupon;
+  couponInfo: CouponInfo;
   onClickStar: () => void;
 }
 
-const CafeInfo = ({
-  cafeName,
-  stampCount,
-  maxStampCount,
-  isFavorites,
-  frontImageUrl,
-  onClickStar,
-}: CafeInfoProps) => {
+const CafeInfo = ({ cafeInfo, couponInfo, onClickStar }: CafeInfoProps) => {
+  const { name: cafeName, isFavorites } = cafeInfo.cafeInfo;
+  const { stampCount, maxStampCount, frontImageUrl } = couponInfo;
```

Comment:
이름이 중복되는 부분이 있네요..! 이 코드가 될까요 ?
```suggestion
const CafeInfo = ({ cafeInfo: { cafeInfo }, couponInfo, onClickStar }: CafeInfoProps) => {
  const { name: cafeName, isFavorites } = cafeInfo;
  const { stampCount, maxStampCount, frontImageUrl } = couponInfo;
```

Reply:
메인 화면에서 카페 정보를 보여주기 위함인데 `CouponInfo`에는 존재하지 않는 데이터인 `CafeName`, `isFavorites`를 사용하기 위해 `Coupon`의 `cafeInfo`를 사용하는 부분입니다.

 윤생이 피드백한 것처럼 구조분해 할당을 했다면 한결 보기 편하게 사용할 수 있었네요. 구조분해할당 완료하였습니다. 네이밍은 api 명세상 합의된 부분이기 때문에 데이터를 직관적으로 이해하기는 어려울 수 있습니다.

[43e44c6](https://github.com/woowacourse-teams/2023-stamp-crush/pull/712/commits/43e44c65ecb609710abf3711241643101f3faf7e)